gradle.properties

```diff
@@ -19,3 +19,5 @@ org.gradle.jvmargs=-Duser.country=US -Duser.language=en
 scalaVersions=2.11.12,2.12.13,2.13.4
 defaultScalaVersions=2.13.4
 runOnceTasks=clean,release
+org.gradle.java.installations.auto-download=false
+org.gradle.java.installations.fromEnv=JDK8,JDK11,JDK17
```

Comment:
What's the behavior when this property is unset?

Reply:
Tries for auto detection: https://docs.gradle.org/current/userguide/toolchains.html#sec:auto_detection